packages/html/src/radio/tests/radio-group.tsx

```diff
@@ -0,0 +1,87 @@
+import { RadioGroup, RadioItem, RadioButtonLabel, RadioButtonInsideLabel, RadioButtonNormal } from '../../radio';
+
+const styles = `
+    #test-area {
+        grid-template-columns: repeat(3, 1fr);
+    }
+`;
+
+export default () => (
+    <>
+        <style>{styles}</style>
+        <div id="test-area" className="k-d-grid">
+            <span>Normal - vertical</span>
+            <span>With Label - vertical</span>
+            <span>Inside Large - vertical</span>
```

Comment:
I believe here is a typo -> should be `Inside Label` (instead of `Inside Large`)